spec/support/api_url_helper.rb

```diff
@@ -271,14 +271,20 @@ def stub_v2_tag_layout_templates(templates)
     end
 
     # Builds the basic v2 tube finding query.
-    def stub_v2_tube(tube, stub_search: true, custom_query: nil, custom_includes: nil)
+    def stub_v2_tube(tube, stub_search: true, custom_query: nil, custom_includes: false) # rubocop:todo Metrics/AbcSize
```

Comment:
I tend to use rubocop:disable / enable over todo if I am happy with the code and have no intention of coming back to it(!)